### PR TITLE
[UI]Added Cookies for banner

### PR DESCRIPTION
**Pull request by Harshit-Dandriyal**
Signed-off-by: Harshit Dandriyal <56961488+Harshit-Dandriyal@users.noreply.github.com>

**Description**
Created cookies for previous Banner
This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,12 +24,15 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@emotion/react": "^11.9.0",
+    "@emotion/styled": "^11.8.1",
```
Comment (Nikhil-Ladha) [inline package.json:28]: Why do we need these 2 packages and the one below `@mui/material` for?

**Comment by l5io** [comment]
🚀 Preview for commit 19de4f1c3cae0abe458b6e2e7f00929f60b4f57c at: https://629fa7114d130d0e1bda059e--layer5.netlify.app

File: package.json (hunk 1 above)
Comment (Harshit-Dandriyal) [reply package.json:28]: Actually, I was working on a dark theme, yes but don't need it in this PR.

**Comment by Harshit-Dandriyal** [comment]
@Nikhil-Ladha Here our cookies are being retrieved after loading the components, It is not working as expected.

**Comment by Nikhil-Ladha** [comment]
Here the first load takes some good time, later it is faster.
Also, one major drawback is that it is not rendering at all, though again there can be some initial render but again it would be the same as we have now.

**Comment by Nikhil-Ladha** [comment]
I think we can let it be as it is now, until and unless anyone manually does a couple of reloads on the home page they would never see that issue, and highly doubt that situation will ever occur (Maybe 1 in a 100k or so?)

**Pull request closed without merging**